tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestDtdData.java

```diff
@@ -222,6 +223,9 @@ private void checkEmpty(Multimap<String, String> m, DtdType type, Element elemen
             case distinguished:
                 distAttributes.add(attribute);
                 break;
+            case metadata:
+                metadataAttributes.add(attribute);
+                break;
```

Comment:
This was previously missed in the switch statement. We don't do anything with these currently, but could.